.github/workflows/pkgci_shark_ai.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       matrix:
         include:
           - name: cpu
-            runs-on: azure-cpubuilder-linux-scale
+            runs-on: ubuntu-latest
```

Comment:
Pin dependencies, including runner types. Use `ubuntu-24.04` here instead of `ubuntu-latest`